Ginger/Ginger/UserControlsLib/UCListView/UcListView.xaml.cs

```diff
@@ -164,18 +164,25 @@ private void ObjListPropertyChanged(object sender, System.ComponentModel.Propert
 
         private void CollectionChangedMethod(object sender, NotifyCollectionChangedEventArgs e)
         {
-            this.Dispatcher.Invoke(() =>
+            try
```

Comment:
in which secnario this method throw exception?

Reply:
added this try catch just to debug the issue which i was facing to see if the exception occurs or not. but the exception was not raised from here.

Reply:
so please remove the try-catch